packages/core/src/Dialog/Dialog.stories.args.tsx

```diff
@@ -0,0 +1,74 @@
+import { ArgTypes } from '@storybook/react'
+import React from 'react'
+import type { IDialogProps } from '.'
+import { Button, colorSchemeNames, Grid, Text } from '..'
+import { DialogSizes } from './Dialog.styled'
+
+export const argTypes: Partial<ArgTypes<IDialogProps>> = {
+  size: {
+    control: { type: 'select' },
+    options: DialogSizes,
+  },
+  triggerNode: {
+    control: { type: 'none' },
+  },
+  children: {
+    control: { type: 'none' },
+  },
+  headerColorScheme: {
+    control: { type: 'select' },
+    options: colorSchemeNames,
+  },
+  headerContent: { type: 'string' },
+  headerIcon: { type: 'boolean' },
+  headerShowCloseButton: { type: 'boolean' },
+  sheet: {
+    control: { type: 'boolean' },
+  },
+  showCloseButton: {
+    control: { type: 'boolean' },
+  },
+  fullWidth: {
+    control: { type: 'boolean' },
+  },
+  hugColor: {
+    control: { type: 'select' },
+    options: [
+      'primary.base',
+      'primary.light',
+      'primary.dark',
+      'secondary.base',
+      'secondary.light',
+      'secondary.dark',
+    ],
+  },
+}
+
+export const defaultArgs: Partial<IDialogProps> = {
+  ariaDescription: 'This is a description',
+  ariaTitle: 'This is a title',
+  children: (
+    <Grid p={[3, 3, 3, 3, '24px', '24px']} gap={3}>
+      <Text>
+        The quick brown fox jumps over the lazy dog to discover new deals every day. Lorem ipsum dolor sit
+        amet, consectetur adipiscing elit.
+      </Text>
+      <Text>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras id erat nec ante semper eleifend. In ex
+        mi, eleifend non ultricies sed, ullamcorper a sem. Aenean mollis mi eget libero commodo, non auctor
+        dui fermentum. Aliquam ultrices nisi in odio sodales, vel feugiat libero gravida.
+      </Text>
+    </Grid>
+  ),
+  scrimDismiss: true,
+  scrimOpacity: 0.25,
+  sheet: false,
+  showCloseButton: true,
+  size: 'md',
+  triggerNode: <Button>Open Dialog</Button>,
+  headerColorScheme: 'neutralDarkOnLightest',
+  headerShowCloseButton: false,
+  headerIcon: false,
+  fullWidth: false,
+  // headerContent: 'Hello World',
```

Comment:
intentional?

Reply:
Unintentional, this is now handled inside the `exampleHeaderProps` within the story file